Basic/ClientDriven/Assets/Scripts/ClientPlayerMove.cs

```diff
@@ -36,60 +44,55 @@ public override void OnNetworkSpawn()
         enabled = IsClient;
         if (!IsOwner)
         {
-            m_Camera.gameObject.SetActive(false);
             enabled = false;
             return;
         }
 
-        m_CharacterController = GetComponent<CharacterController>();
-    }
-
-    [ClientRpc]
-    public void SetSpawnClientRpc(Vector3 position, ClientRpcParams clientRpcParams = default)
-    {
-        m_CharacterController.enabled = false;
-        transform.position = position;
-        m_CharacterController.enabled = true;
-        gameObject.SetActive(true);
-    }
+        m_PlayerInput.enabled = true;
+        m_ThirdPersonController.enabled = true;
 
-    // DOC START HERE
```

Comment:
Was the doc updated?

Reply:
Right. If it makes sense to you, I've created a task to update ClientDriven doc & create a PR with the changes here: [MTT-5106](https://jira.unity3d.com/browse/MTT-5106)

Reply:
When is that fix gonna land? Cause as soon as we release this PR, docs will break.

Reply:
Created this [docs PR](https://jira.unity3d.com/browse/MTT-5013) with scripting changes!

Reply:
you mean this? https://github.com/Unity-Technologies/com.unity.multiplayer.docs/pull/865/files